src/Domain/ValueObject/ContactMedia.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Domain\ValueObject;
+
+use BBC\ProgrammesPagesService\Domain\Enumeration\ContactMediaEnum;
+
+class ContactMedia
+{
+    private $value;
+    private $freetext;
+    private $type;
+
+    public function __construct($type, $value, $freetext)
```

Comment:
Typehinting?
Should you check the type is valid?

Reply:
Yeah, good point. Currently and checking that in the src/Domain/Entity/ContactMediaMap.php (which is orchestring the creation of ContactMedia), but I agree with you, it should be here.